### PR TITLE
Feat 0.13

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*
```
Comment (Meldiron) [inline .dockerignore:1]: This scares me 😅 Are we sure we can ignore everything? Do I miss-understand it?

File: README.md [hunk 2]
```diff
@@ -8,32 +8,51 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Ruby
...
-This playground doesn't include all of Ruby best practices but rather intended to show some of the most simple examples and use cases of using the Appwrite API in your Ruby application. Rubocop is enforced and the configuration can be found at [.rubocop.yml](.rubocop.yml)
+This playground does not adhere to Ruby best practices but rather is intended to show some of the most simple examples and use cases of using the Appwrite API in your Ruby application. Rubocop is enforced and the configuration can be found at [.rubocop.yml](.rubocop.yml)
 
-## System Requirements 
-* A system with Ruby installed.
-* You have readily available AppWrite running instance (localhost in most cases).
-* Create a project in AppWrite instance using console.
-* Generate a secret key in the AppWrite instance using console.
+## Requirements 
+* A system with Ruby 2.7+ or Docker installed.
```
Comment (Meldiron) [inline README.md:14]: Can we apply same copy from sdk-for-python for requirements and instlaation? I will also try to stay consistent in other playgrounds

**Review by Meldiron — CHANGES_REQUESTED** [review]
Left a few notes

File: .dockerignore (hunk 1 above)
Comment (abnegate) [reply .dockerignore:1]: This ignores everything but the following lines negate it to include the required files, so all files in `lib` and any filename beginning with `Gemfile` will be available in the build context but nothing else

File: README.md [hunk 3]
```diff
@@ -8,32 +8,52 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Ruby
...
-* Add Document
-* List Documents
-* Upload File
-* Create User
-* List User
+### API's Covered:
+
+- Databse
+    * Create Collection
+    * List Collection
```
Comment (Meldiron) [inline README.md:36]:
```suggestion
    * List Collections
```

File: README.md [hunk 4]
```diff
@@ -8,32 +8,52 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Ruby
...
+    * Add Document
+    * List Documents
+    * Delete Document
+    * Delete Collection
+
+- Storage
+    * Create Bucket
```
Comment (Meldiron) [inline README.md:43]: Do we have list buckets?

File: README.md [hunk 5]
```diff
@@ -8,32 +8,52 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Ruby
...
+    * Upload File
+    * List Files
+    * Delete File
+
+- Users
+    * Create User
+    * List User
```
Comment (Meldiron) [inline README.md:50]:
```suggestion
    * List Users
```

**Review by Meldiron — APPROVED** [review]
LGTM

**Pull request merged**